vere/unix.c

```diff
@@ -590,7 +590,8 @@ _unix_watch_file(u3_ufil* fil_u, u3_udir* par_u, c3_c* pax_c)
 
   fil_u->dir = c3n;
   fil_u->dry = c3n;
-  fil_u->pax_c = pax_c;
+  fil_u->pax_c = c3_malloc(1 + strlen(pax_c));
+  strcpy(fil_u->pax_c, pax_c);
```

Comment:
If I understand `strcopy()` correctly, this will work properly as long as the input `pax_c` is null-terminated. Are we sure that's always the case? Otherwise the last bit will just be whatever `malloc()` produces.

Reply:
`pax_c` always comes from `_unix_down()`, which explicitly terminates it.